devfiles/python-django/devfile.yaml

```diff
@@ -77,3 +77,11 @@ commands:
             ]
           }]
         }
+    - 
+      name: set up venv
```

Comment:
When I tested this, the devfile editor in the Getting Started area would complain about this being after the `debug` task. When I moved it before the `debug` task, all was well. :man_shrugging:

Reply:
Actually, the indentation is wrong. YAML is crap!